src/scripts/build_summarization_dataset.py

```diff
@@ -0,0 +1,20 @@
+import argparse
```

Comment:
Det her er ikke et modul, men en modul-docstring er stadig rigtig god stil. Når det kommer til scripts, kan man derudover også skrive hvordan man kalder scriptet:

```suggestion
"""Script that builds the summarisation dataset.

Usage:
    >>> python src/scripts/build_summarization_dataset.py
    
    We can also specify the total number of articles, that we want to process (default is 500):
    >>> python src/scripts/build_summarization_dataset.py --total_articles 100
""""

import argparse
```